src/CoreBundle/Service/RequestInfoServiceInterface.php

```diff
@@ -38,12 +38,19 @@ public function getChameleonRequestType();
     public function isChameleonRequestType($requestType);
 
     /**
-     * Returns true if the current request is a backend request AND the current page is the template engine.
+     * Returns true if the current request is a frontend request and the current page is the template engine.
```

Comment:
Both of these comments are a bit confusing, can we make clearer what the method does?

Reply:
reworded and added usage so it should clearer what's going on.